opm/simulators/flow/EclWriter.hpp

```diff
@@ -498,7 +498,7 @@ class EclWriter : public EclGenericWriter<GetPropType<TypeTag, Properties::Grid>
         const auto waterActive = FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx);
         const auto enableSwatinit = simulator_.vanguard().eclState().fieldProps().has_double("SWATINIT");
         const auto opm_rst_file = Parameters::Get<Parameters::EnableOpmRstFile>();
-        const auto read_temp = enableEnergy || (opm_rst_file && enableTemperature);
+        const auto read_temp = enableEnergy;
```

Comment:
This change means that the `opm_rst_file` object created on the line above becomes unused.  Is that intentional?  If so, please remove the `opm_rst_file` object.  If not, then please use it where it's needed.